ceph.spec.in

```diff
@@ -109,15 +114,29 @@ BuildRequires:	selinux-policy-devel
 %endif
 %if 0%{with make_check}
 %if 0%{?fedora} || 0%{?rhel}
+%if 0%{with python2}
 BuildRequires:	python-cherrypy
+BuildRequires:	python-pecan
 BuildRequires:	python-werkzeug
+%else
+BuildRequires:	python%{python3_pkgversion}-cherrypy
```

Comment:
can we rename `python3_pkgversion` to something like `python_buildid`, and make it an empty string if `with python2`, and `3` otherwise?. and always "BuildRequire: python%{python_buildid}-foobar" instead of duplicating the package names with and without this postfix.

Reply:
OK